### ISSUE TITLE
Lots of problems

### ISSUE THREAD

**Issue by MasterIO02** [mekanism/Mekanism#5545]
Hi everyone, I got a lot of problems with mekanism on my server.
First, i get 3-4 logs a second saying to me : 

`[22:12:45]: [WARN] [Mekanism] 'TransitRequest' was wrapped around a non-IItemHandler inventory. This should not happen!
[22:12:45]: [WARN] [Mekanism] - null tile`

Before, it said the position of the machine but here it says "null tile".

I got a problem with the Digital Miner. When we broke the Digital Miner with the Atomic Disassembler, it dropped 12 items.

I got a problem also with "Bounding Blocks". Everytime we brake a mekanism block, a "bounding block" spawn and it is unbreakable even in Creative. I used WorldEdit for deleting those blocks.

Version : 
Forge: 1.12.2_2768 
Mekanism: 1.12.2-9.7.8.379

Sorry for my bad english, and thanks for the people that will help me :)

**Comment by thiakil** [comment]
Transit request is the same as #5534 

>  it dropped 12 items

12 digital miners? or something else?
This + the bounding blocks issue points to an issue with another mod; the bounding blocks will always be broken by the breaking of the block itself.

Please provide a FULL debug log.

**Comment by MasterIO02** [comment]
Yes the digital miner drops 12 digital miners when we broke it, it dont break entirely but block per block.
If this is another mod that causes that + the bounding block, do you have an idea of what mod is it?

I'll try myself to delete mod per mod for testing.

My log file is over 2GB because spam of the "transit request". There is nothing interesting in it xD

**Comment by thiakil** [comment]
> If this is another mod that causes that

yes its definitely another mod (its not normal behaviour), though I can't say what given there are thousands of mods out there

> My log file is over 2GB because spam of the "transit request". There is nothing interesting in it xD

I still want to see it. You could remove the lines that are just the transit request message. Debug logs are always huge

**Comment by MasterIO02** [comment]
Ok so I found a Logistical Sorter with nothing around, i broke it and now I dont have logs.
I disabled the Digital Miner since persons on my server were using it to duplicate items in the inventory of the Digital Miner.

I still have spam from ic² because of Mekanism, but i dont think it's Mekanism that causes this spam : https://pastebin.com/AYbqLsYB

I noticed that when nobody is in the server this spam begin, and when somebody connects it stops.
